demo-v1/src/components/screens/KeypadScreen.tsx

```diff
@@ -21,22 +21,42 @@ export const validateKeypadInput = (value: string) => {
   return /^[0-9+*#]*$/.test(value);
 };
 
+const validatePhoneNumber = (value: string) => {
```

Comment:
```suggestion
const isValidPhoneNumber = (value: string): boolean => {
```

Reply:
The return value here is inferred, I can change the name however

Reply:
On second thought, `isValidPhoneNumber` seems too similar to `isDialNumberValid`